src/components/Table/simpleHelpers.ts

```diff
@@ -0,0 +1,36 @@
+import { GridDataRow } from "src/components/Table/GridTable";
+
+/** A helper for making `Row` type aliases of simple/flat tables that are just header + data. */
+export type SimpleHeaderAndDataOf<T> = { kind: "header" } | ({ kind: "data" } & T);
```

Comment:
This file is extracted from `GridTable.ts`, no changes, just trying to break the file up a bit. Should do more in a follow up PR.